src/Cake.Common/Tools/OctopusDeploy/OctopusDeployAliases.cs

```diff
@@ -1,125 +1,149 @@
+        ///
+        ///         IgnoreExisting = true // if this release number already exists, ignore it
+        ///     });
+        /// </code>
+        /// </example>
+        [CakeMethodAlias]
+        public static void OctoCreateRelease(this ICakeContext context, string projectName, CreateReleaseSettings settings)
+        {
+            if (context == null)
+            {
+                throw new ArgumentNullException(nameof(context));
+            }
+
+            var packer = new OctopusDeployReleaseCreator(context.FileSystem, context.Environment, context.ProcessRunner, context.Tools);
+            packer.CreateRelease(projectName, settings);
+        }
+
+        /// <summary>
+        /// Pushes the specified package to the Octopus Deploy repository
+        /// </summary>
+        /// <param name="context">The cake context</param>
+        /// /// <param name="server">The Octopus server URL</param>
+        /// <param name="apiKey">The user's API key</param>
+        /// <param name="packagePath">Path to the package</param>
+        /// <param name="settings">The settings</param>
+        [CakeMethodAlias]
+        public static void OctoPush(this ICakeContext context, string server, string apiKey, FilePath packagePath, OctopusPushSettings settings)
+        {
+            OctoPush(context, server, apiKey, new[] { packagePath }, settings);
+        }
+
+        /// <summary>
+        /// Pushes the specified packages to the Octopus Deploy repository
+        /// </summary>
+        /// <param name="context">The cake context</param>
+        /// <param name="server">The Octopus server URL</param>
+        /// <param name="apiKey">The user's API key</param>
+        /// <param name="packagePaths">Paths to the packages</param>
+        /// <param name="settings">The settings</param>
+        [CakeMethodAlias]
+        public static void OctoPush(this ICakeContext context, string server, string apiKey, IEnumerable<FilePath> packagePaths, OctopusPushSettings settings)
+        {
+            if (context == null)
+            {
+                throw new ArgumentNullException(nameof(context));
+            }
+
+            if (packagePaths == null)
+            {
+                throw new ArgumentNullException(nameof(packagePaths));
+            }
+
+            var pusher = new OctopusDeployPusher(context.FileSystem, context.Environment, context.ProcessRunner, context.Tools);
+            pusher.PushPackage(server, apiKey, packagePaths.ToArray(), settings);
+        }
+
+        /// <summary>
+        /// Packs the specified folder into an Octopus Deploy package
+        /// </summary>
+        /// <param name="context">The cake context</param>
+        /// <param name="id">The package ID.</param>
+        /// <param name="settings">The settings</param>
+        [CakeMethodAlias]
+        public static void OctoPack(this ICakeContext context, string id, OctopusPackSettings settings = null)

```

Comment:
Could we create an overload for this method (without the settings) instead of using default values. We've seen some funny things happening with older versions of Roslyn when doing this.